fastify.js

```diff
@@ -394,6 +405,14 @@ function build (options) {
       throw new Error(`Missing handler function for ${opts.method}:${opts.url} route.`)
     }
 
+    var jsonBodyLimit = _fastify._jsonBodyLimit
+    if (opts.jsonBodyLimit !== undefined) {
+      if (!Number.isInteger(opts.jsonBodyLimit)) {
+        throw new TypeError(`'jsonBodyLimit' option must be an integer. Got: '${opts.jsonBodyLimit}'`)
```

Comment:
I would add a `typeof` after "Got".
`Got: '${typeof opts.jsonBodyLimit}'`

Reply:
With [this suggestion](https://github.com/fastify/fastify/pull/627#discussion_r160046782) from @StarpTech, users will need to see the actual value to understand why it wasn't valid.